### PR TITLE
Add sent and received status to outgoing messages

### PR DESCRIPTION
**Pull request by Flaburgan**
This PR cleans the front-end for outgoing message status. `message.IsSent` and `message.Receipt` just have to be set correctly by the back for the status to be displayed.

image

### CODE REVIEW

**Comment by Flaburgan** [comment]
To be closer than the official Signal client, I moved the metadata in the message bubbles. It was also easier for the dark theme, though not enough so I had to do two versions of the icons.

image

image

File: axolotl-web/src/pages/MessageList.vue [hunk 1]
```diff
@@ -22,91 +22,84 @@
         </div>
           <div v-for="(message) in messageList.Messages.slice().reverse()"
               :class="{'col-12':true,
-                      'sent':message.Outgoing,
-                      'reply':!message.Outgoing,
+                      'outgoing': message.Outgoing,
+                      'sent':message.IsSent && message.Outgoing,
+                      'received':message.Receipt && message.Outgoing,
```
Comment (nanu-c) [inline axolotl-web/src/pages/MessageList.vue:27]: This should be message.IsRead because Receipt is not used at all

**Review by nanu-c — CHANGES_REQUESTED** [review]

File: axolotl-web/src/pages/MessageList.vue (hunk 1 above)
Comment (Flaburgan) [reply axolotl-web/src/pages/MessageList.vue]: Fixed.

**Comment by Flaburgan** [comment]
OK I now use isRead, and I changed the error rendering too. This is ready for review. If you pulled that branch you need to hard reset it as I force pushed it (sorry, habits. I won't do that anymore).

**Review by nanu-c — APPROVED** [review]

**Pull request merged**